src/multi-select/dropdown.tsx

```diff
@@ -65,6 +68,16 @@ const DropdownHeading = css({
   },
 });
 
+const ClearSelectedButton = css({
+  cursor: "pointer",
+  background: "none",
+  border: 0,
+  padding: 0,
+  "[hidden]": {
+    display: "none",
+  },
```

Comment:
I think instead of this maybe conditionally rendering might be better for screen readers
```tsx
{contentProps["value"].length && <button>X</button>}
```

Reply:
Added conditional rendering